### PR TITLE
medical vendor additions

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request

Gives medical venders 6 metal injects, 6 sal acid injects 2 boxes of body bags, 1 box of latex gloves and face masks
Adds 12 penlights to the medical vender as well

## Why It's Good For The Game

The boxes of body bags go fast if you use them with the intent of burning the body in the bag or to put in a morgue (in some maps)
Gloves and masks are QoL for people that late join or respawn after dusting a few times
Extra injectors give a bit more use for personal money

Penlights are amazing.

## Changelog
:cl:
add: Added new things to the medical vender, boxes of body bags, gloves, masks, hypo sprays, penlights ect
/:cl:

### CODE REVIEW

**Comment by Trilbyspaceclone** [comment]
Thats a sad!
I wounder if revival surgery is on the table or not tho

**Comment by Trilbyspaceclone** [comment]
> No revival surgery for the same reason. Death should be final, and revival methods mean any abnorm that can't gib is completely invalidated.

Understood, and completely reasonable! I just wish their was more time to save people V_V

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
Remove the gloves, you get those from Cargo

File: code/modules/vending/medical.dm [hunk 1]
```diff
@@ -15,7 +16,12 @@
 					/obj/item/healthanalyzer/wound = 4,
 					/obj/item/stack/medical/ointment = 2,
 					/obj/item/stack/medical/suture = 2,
-					/obj/item/stack/medical/bone_gel = 4)
+					/obj/item/stack/medical/bone_gel = 4,
+					/obj/item/reagent_containers/hypospray/medipen/mental = 6,
+					/obj/item/reagent_containers/hypospray/medipen/salacid = 6,
+					/obj/item/storage/box/bodybags = 4,
+					/obj/item/storage/box/gloves = 1,
```
Comment (Trilbyspaceclone) [inline code/modules/vending/medical.dm:23]:
```suggestion
```

**Review by Kitsunemitsu — APPROVED** [review]
Approved/

**Pull request merged**